bin/gobblin-mapreduce.sh

```diff
@@ -4,6 +4,9 @@
 ############### Run Gobblin Jobs on Hadoop MR ################
 ##############################################################
 
+# Set during the distribution build
+GOBBLIN_VERSION=@project.version@
```

Comment:
@sahilTakiar thanks for your inputs, this definitely makes it more robust. Fixed.